### DISCUSSION TITLE
feedbacks needed to improve

### DISCUSSION THREAD

**Discussion by zxstim** [kaiachain/kaia-dapp-mono#55] [General]
hi all,

to further improve `kaia-dapp-mono` as the repo for building dapps on kaia, we need some feedbacks/suggestions/ideas/comments.

keep it real, just comment whatever you think, keep it civilized tho (no slurs, racist, inappropriate comments). Do note that if you want to complain, we expect you to give us some suggestions to fix.

**Comment by anudeep11k** [comment]
Suggestion: Creating a dedicated section for TheGraph significantly enhances its utility for developers and simplifies their experience.

**Reply by zxstim** [reply]
maybe i will add a data track

**Reply by anudeep11k** [reply]
Thats a big plus +
